apps/prairielearn/python/prairielearn/core.py

```diff
@@ -1657,7 +1644,7 @@ def string_to_2darray(
 
 
 def latex_from_2darray(
-    A: numbers.Number | npt.NDArray[Any],
+    A: numbers.Number | _NumPyScalarType | npt.NDArray[Any],
```

Comment:
I added `_NumPyScalarType` for consistency with other functions that have similar signatures. https://github.com/PrairieLearn/PrairieLearn/pull/11294/files#r1941808269 is probably relevant; we should consider making `numbers.Number` a part of `_NumPyScalarType`.

Reply:
I changed around how these types work here: 1d83004